### PR TITLE
[iOS] Fix conditions for tabBarSelectedLabelColor & selectedButtonColor

### PR DESCRIPTION
**Pull request by datso**
Without that fix it is not possible to show original iconImageSelected and also tabBarSelectedLabelColor uses invalid condition.

### CODE REVIEW

File: ios/RCCTabBarController.m [hunk 1]
```diff
@@ -166,6 +166,10 @@ - (instancetype)initWithProps:(NSDictionary *)props children:(NSArray *)children
     id selectedIcon = tabItemLayout[@"props"][@"selectedIcon"];
     if (selectedIcon) {
       iconImageSelected = [RCTConvert UIImage:selectedIcon];
+      
+      if (!selectedButtonColor) {
```
Comment (guyca) [inline ios/RCCTabBarController.m]: Can you please clarify this change?

Comment (datso) [inline ios/RCCTabBarController.m]: I can't set an original icon for selected state without UIImageRenderingModeAlwaysOriginal. Without this condition, selectedIcon are always colored with the default (light-blue) color.

Comment (guyca) [inline ios/RCCTabBarController.m]: Sorry to trouble you, Can you please include code samples to help me understand the flow this fixes?

Comment (datso) [reply ios/RCCTabBarController.m]: Sorry for long delay, without this fix it is not possible to show selected icon as is.

Without fix:
without_fix
With fix:
with_fix

Comment (datso) [reply ios/RCCTabBarController.m]: The same for tabBarSelectedLabelColor.

Comment (datso) [reply ios/RCCTabBarController.m]: @guyca

Comment (krystofcelba) [reply ios/RCCTabBarController.m]: Please merge this. I have the same problem.

**Pull request closed without merging**